### DISCUSSION TITLE
General Project Discussion

### DISCUSSION THREAD

**Discussion by PinkSylvie** [uprm-inso4101-2023-2024-S1/semester-project-tutoring-app#76] [General]
I made this board to get some trackable discussion and activity going on here instead of in discord for grading sake. Staring with the documentation we need to turn in. An issue was created by @print-Eruki for it and I picked it up. I would like to get some discussion going with all teams to write based on what we've worked on. An example or what we need would be [this proposal](https://online.upr.edu/pluginfile.php/3864420/mod_page/content/8/ProposalAnonymized.pdf) 
![image](https://github.com/uprm-inso4101-2023-2024-S1/semester-project-tutoring-app/assets/93553908/2c1f1130-faff-4d46-8d79-c782a8e31107)

**Comment by PinkSylvie** [comment]
@uprm-inso4101-2023-2024-S1/tutoring-app @print-Eruki @G4vp @Yahid1 @panchi64

**Comment by panchi64** [comment]
Sounds great, I'll get to writing up as much as I can from my team as soon as I can today.
